pyproject.toml

```
@@ -36,7 +37,7 @@ exclude = ["tests", "docs"]
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-django = ">=4.2,<5.0"
+django = ">=4.2"
```

Further changes are needed in 5.1 of Django to maintain support - https://github.com/farridav/django-jazzmin/pull/534
```suggestion
django = ">=4.2,<5.1"
```

---

Hmm, if we do this, then people won't even bother installing the app beyond this version, nor will they be able to tolerate or fix minor issues in unsupported versions.

The decision to have an upper bound is contentious I admit, if this package is absolutely broken on Django 5.1 then maybe it's a good idea, but otherwise, maybe we should leave it open and endeavour to resolve issues as they arise

---

Ah yeah good point. It's not totally broken in 5.1. This seems to only impact fieldsets, so likely still a widespread issue.